.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,25 @@
+name: Improve our documentation
+description: Report issues in our documentation or things we could document better.
+labels: ['docs']
```

Comment:
I'm not sure we should add tags this way. I'd rather we add tags as part of grooming. By adding tags now it'll be more difficult to tell which issues are groomed and which are not. Thoughts?

Reply:
That's a fair point. What do you think about adding the "label" as part of the title? would that be helpful at all?

```
[Docs]: <title>
[Feat]: <title>
```

We could also create a new label `needs-triage ` or `ungroomed` or something that we remove from tickets to help ensure we've groomed them.

Reply:
Hmm, I guess "ungroomed" could work.

Reply:
Add ungroomed label https://github.com/dequelabs/axe-core/labels?q=ungroomed